description: Disable IPv6 pinging of real server in the background

--- a/net/dns/host_resolver_impl.cc
+++ b/net/dns/host_resolver_impl.cc
@@ -78,9 +78,9 @@ const unsigned kMinimumTTLSeconds = kCacheEntryTTLSeconds;
 // cached.
 const int kIPv6ProbePeriodMs = 1000;

-/* RIPE NCC k.root-servers.net. 2001:7fd::1 (anycasted) */
+/* Loopback address */
 const uint8_t kIPv6ProbeAddress[] =
-    { 0x20, 0x01, 0x07, 0xfd, 0x00, 0x00, 0x00, 0x00,
+    { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
       0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01 };

 // We use a separate histogram name for each platform to facilitate the

